core/src/test/scala/org/apache/spark/benchmark/Benchmarks.scala

```diff
@@ -78,8 +99,12 @@ object Benchmarks {
           // Maven build
           targetDirOrProjDir.getCanonicalPath
         }
+        // Force GC to minimize the side effect.
+        System.gc()
         runBenchmark.invoke(null, Array(projDir))
```

Comment:
How about to continue if a benchmark crashes, and complete others. I would imagine an use cases when I run a few benchmarks during night, and re-run only crashed benchmarks in the morning.

Reply:
Make sense.

Reply:
Yeah .. sounds good

Reply:
Let me do it later though after checking if it is able to run in one go. I would like to make sure it passes at least once.